### PR TITLE
Update B&R icon

### PR DESCRIPTION
**Pull request by adamrusted**
bandrautomation

**Issue:** contributes toward #9072

**Similarweb rank:** [174,642](https://www.similarweb.com/website/br-automation.com/#overview)

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
JSON update not necessary at this time.

### CODE REVIEW

File: icons/bandrautomation.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>B&amp;R Automation</title><path d="M24 16.444H0v2.088h24v-2.088Zm-8.812-1.59-1.418-1.766.623-.767-.831-1.036-.623.767-1.565-1.959 2.192.008V8.592H9.39v1.405l.667.835c-.072.036-.538.245-.691.349a2.2 2.2 0 0 0-.939 1.18 2.046 2.046 0 0 0 1.04 2.349c.823.417 1.931.333 2.758-.04.168-.081.333-.177.482-.289l.124-.093.458.566h1.899ZM6.179 9.989a2.108 2.108 0 0 0 1.204-1.984c0-1.541-1.116-2.537-2.991-2.537H0v9.39h4.549c2.111 0 3.123-1.216 3.123-2.629a2.251 2.251 0 0 0-1.489-2.24h-.004Zm15.035 1.401s2.132-.442 2.132-2.931c0-1.75-1.076-2.991-3.252-2.991h-4.445v9.39h2.092V11.39h1.004l2.778 3.468h2.473L21.21 11.39h.004Zm-11.073 1.911a.711.711 0 0 1-.044-.831c.209-.261.482-.466.795-.59l.193.236.879 1.117c-.213.14-.434.26-.667.365a1.147 1.147 0 0 1-.975-.133.882.882 0 0 1-.177-.168l-.004.004Zm-5.837-.008H2.088v-2.61h2.256c.911 0 1.453.45 1.453 1.305 0 .803-.518 1.305-1.493 1.305ZM17.745 9.78V7.259h2.212c.92 0 1.494.385 1.494 1.26 0 .799-.538 1.261-1.494 1.261h-2.212ZM2.088 7.062h2.007c.763 0 1.216.401 1.216 1.116 0 .715-.453 1.168-1.32 1.168H2.088V7.062Z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/bandrautomation.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>B&amp;R Automation</title><path d="M0 16.238h24v2.132H0zm9.35-1.673c.82.415 1.921.329 2.743-.041.17-.077.328-.179.48-.289a3.36 3.36 0 0 0 .123-.093l.582.692h1.888l-1.533-1.882.619-.762-.827-1.03-.62.765-1.558-2.031 2.157.006V8.477H9.166l.003 1.4.77.831c-.07.038-.537.245-.689.346-.425.28-.791.675-.937 1.173-.265.91.2 1.916 1.037 2.338zm.633-2.23c.185-.271.498-.449.789-.586l.19.236.872 1.11a4.514 4.514 0 0 1-.664.363c-.307.1-.7.059-.97-.13a.783.783 0 0 1-.175-.169c-.179-.235-.22-.564-.042-.823zm7.672-.999.97.003 2.785 3.508h2.503l-2.8-3.498s2.144-.342 2.144-2.783c0-1.717-1.081-2.936-3.268-2.936h-4.466v9.217h2.132v-3.51zm-.012-3.957h2.08c.883 0 1.434.378 1.434 1.232 0 .782-.516 1.232-1.434 1.232h-2.08V7.379zm-9.879 4.878c0-1.028-.524-1.841-1.509-2.196.787-.368 1.22-1.079 1.22-1.943 0-1.51-1.128-2.488-3.029-2.488H0v9.204h4.603c2.139 0 3.161-1.194 3.161-2.577zM2.132 7.123H4.08c.776 0 1.236.37 1.236 1.023 0 .654-.46 1.071-1.34 1.071H2.131V7.123zm0 6.307v-2.81h2.1c.882 0 1.408.483 1.408 1.405 0 .862-.5 1.404-1.448 1.404h-2.06z"/></svg>
```
I see some differences
image
bandrautomation-comparison

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

File: icons/bandrautomation.svg (hunk 1 above)
Comment (adamrusted) [reply icons/bandrautomation.svg:1]: Not sure what you're comparing to @PratyushJayachandran - the SVG in the website header seems to be a [perfect match](https://github.com/simple-icons/simple-icons/assets/1830867/af36448e-acf5-48eb-a8e6-9d95606c9419) on my end...

Comment (PratyushJayachandran) [reply icons/bandrautomation.svg:1]: I used the logo here https://www.br-automation.com/en/ and tried again. Still seeing the same difference with the logo in PR. Attaching SVG below
logo

Comment (adamrusted) [reply icons/bandrautomation.svg:1]: How strange... @LitoMore - can you chip in on this one? I know we've had a few recently where there have been issues like this...

Comment (PeterShaggyNoble) [reply icons/bandrautomation.svg:1]: I seems the problem here stems from the fact that there are in fact 2 different logos in use on the site! One in the header and one in the country selection modal. When I compare @adamrusted's version with the latter then I see the same differences as @PratyushJayachandran, but when I compare with the former it's a perfect match. I would say the one in the header would be more indicative of their primary branding so we should run with that.

Having said that, there is an [excess point](https://github.com/simple-icons/simple-icons/assets/15157491/c4a7798f-5dc7-456c-867d-1bc7b037e27c) in the cutout of the ampersand which could be removed before merging.

**Pull request merged**